syntax = "proto3";

service Showtime {
    rpc GetShowtimes(EmptySchedule) returns (stream Schedule) {}
    rpc GetShowtimeByDate(Date) returns (Schedule) {}
}

message Schedule {
    string date = 1;
    repeated string movies = 2;
}

message Date {
    string date = 1;
}

message EmptySchedule { }